cms/djangoapps/contentstore/views/transcripts_ajax.py

```diff
@@ -173,26 +173,23 @@ def download_transcripts(request):
             item.location.course_key,
             'subs_{filename}.srt.sjson'.format(filename=filename),
         )
-        sjson_transcript = contentstore().find(content_location).data
+        input_format = Transcript.SJSON
+        transcript_content = contentstore().find(content_location).data
     except NotFoundError:
         # Try searching in VAL for the transcript as a last resort
         transcript = None
         if is_val_transcript_feature_enabled_for_course(item.location.course_key):
-            transcript = get_video_transcript_content(
-                language_code=u'en',
-                edx_video_id=item.edx_video_id,
-                youtube_id_1_0=item.youtube_id_1_0,
-                html5_sources=item.html5_sources,
-            )
+            transcript = get_video_transcript_content(edx_video_id=item.edx_video_id, language_code=u'en')
```

Comment:
Now we don't need youtube id and html5 sources?

Reply:
Exactly!